### PR TITLE
[Merged by Bors] - perf: speed up `Ring.DirectLimit.field` instance

### PR DESCRIPTION
**Pull request by Vierkantor**
This is split off from #7584

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mattrobball** [comment]
bors cancel 

Oops maybe wait for CI :)

**Comment by mattrobball** [comment]
bors d+

**Comment by Vierkantor** [comment]
bors merge

File: Mathlib/Algebra/DirectLimit.lean [hunk 1]
```diff
@@ -593,7 +593,7 @@ theorem of.zero_exact_aux [Nonempty ι] [IsDirected ι (· ≤ ·)] {x : FreeCom
   · rintro x y ⟨j, t, hj, hyt, iht⟩
     rw [smul_eq_mul]
     rcases exists_finset_support x with ⟨s, hxs⟩
-    rcases (s.image Sigma.fst).exists_le with ⟨i, hi⟩
+    rcases(s.image Sigma.fst).exists_le with ⟨i, hi⟩
```
Comment (Vierkantor) [inline Mathlib/Algebra/DirectLimit.lean:596]:
```suggestion
    rcases (s.image Sigma.fst).exists_le with ⟨i, hi⟩
```

**Comment by Vierkantor** [comment]
Come on, bors!

bors r+

**Pull request closed without merging**